--- a/qga/vss-win32/install.cpp
+++ b/qga/vss-win32/install.cpp
@@ -544,9 +544,10 @@ STDAPI COMUnregister(void)
         }
     }
     return hr;
 }
 
 
+#if !defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 14
 /* Helper class for _bstr_t */
 namespace _com_util
 {
@@ -562,9 +563,10 @@ namespace _com_util
         wcscpy(bstr, wstring);
         delete[] wstring;
         return bstr;
     }
 }
+#endif /* __MINGW64_VERSION_MAJOR < 14 */
 
 
 /* Stop QGA VSS provider service from COM+ Applications */
